docs/01-start-here/01-installation-steps.md

```diff
@@ -41,13 +41,15 @@ Prerequesites:
 5. Run the container (dev service). The first time you do so the container image would be pulled from the ECR.
     - ./dev.sh
 6. Build and run the app within the container
-    - `make install compile && ./sbt`
+    - `cd frontend`
```

Comment:
This shouldn't be necessary. Let me find out why it is not starting up in /frontend